tests/cmake_integration/application/deps/HighFive

```diff
@@ -0,0 +1 @@
+/home/lucg/git/bbp/HighFive/tests/cmake_integration/../..
```

Comment:
errr.... Pretty sure I don't have that directory!

Reply:
Yes, it's automatically regenerated when the script that checks everything runs. Needs to be removed and added to `.gitignore`.